packages/sitecore-jss/src/layout/rest-layout-service.ts

```diff
@@ -186,25 +186,31 @@ export class RestLayoutService extends LayoutServiceBase {
       ? this.serviceConfig.dataFetcherResolver<LayoutServiceData>(req, res)
       : this.getDefaultFetcher<LayoutServiceData>(req, res);
 
-    return fetchRouteData(itemPath, { fetcher, ...fetchOptions }).catch((error) => {
-      if (error.response?.status === 404) {
-        // Aligned with response of GraphQL Layout Service in case if layout is not found.
-        // When 404 Rest Layout Service returns
-        // {
-        //   sitecore: {
-        //     context: {
-        //			   pageEditing: false,
-        //				 language
-        //		 },
-        //		 route: null
-        //	 },
-        // }
-        //
-        return error.response.data;
-      }
-
-      throw error;
-    });
+    return fetchRouteData(itemPath, { fetcher, ...fetchOptions })
+      .then((data: LayoutServiceData) => {
+        data.sitecore.tracked = this.serviceConfig.tracking !== false;
```

Comment:
Just to clarify: why do we assume that the page was tracked if the tracking property is undefined?

Reply:
The default value of the tracked property is 'true'.

Reply:
The default value of the `tracking` property is `true`.